contributing/data-model-schema.txt

```diff
@@ -29,9 +29,32 @@ wastes your time. From the top-level :file:`bioformats` folder, while
 the most thorough is :command:`git clean -dfx`.
 
 
+Major or minor release?
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A minor release of the OME model schema may suffice for changes like
```

Comment:
I am currently working on a BF docs PR which will also reference it so happy to have this become more widely used (it isn't referenced as yet but I think Roger has already used it for the version requirements page wording - I certainly have https://trello.com/c/HSPvUKRK/474-best-practice-for-imperatives-key-word-usage on the docs board after he pointed it out to me)